docs/modules/mysql.md

```
@@ -0,0 +1,165 @@
+
+#### Container Methods
+
+| Method                  | Return Type | Description                            |
+|-------------------------|-------------|----------------------------------------|
+| `getConnectionString()` | `String`    | Returns a `mysql://` connection URI    |
+| `stop()`                | `Void`      | Stops the container (10 s timeout)     |
+| `delete()`              | `Void`      | Removes the container                  |
+
+##### Connection string format
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)